gsy_framework/community_datasheet/community_datasheet_parser.py

```diff
@@ -105,6 +105,8 @@ def _create_grid(self) -> Dict:
         grid = []
         for member_name, assets in self._datasheet.assets_by_member.items():
             home_representation = self._create_home_representation(member_name)
+            self._datasheet.members[member_name]["uuid"] = home_representation["uuid"]
```

Comment:
NIT: If that's possible I would change the "uuid" name to "home_uuid".

Reply:
I will omit this comment because it is highly debatable whether the term home is correct here. The term "member" is the correct one in this context, however I believe this is a name duplication here.